src/screens/Episode/helpers/getTime.ts

```diff
@@ -0,0 +1,17 @@
+import * as dateFns from 'date-fns'
+import { DateFormatType, getFormattedDate } from 'src/helpers'
+
+const getTime = (amntOfSeconds: number): string => {
+  const HHmmss = getFormattedDate(
```

Comment:
```suggestion
  const formattedDate= getFormattedDate(
```

Reply:
"formattedDate"... i will try to find better variant

Reply:
I think it's logic that `getFormattedDate` function returns `formattedDate`

Reply:
> "formattedDate"... i will try to find better variant, because it's not quite a date, it is time in "hh:mm:ss" format

I think it's logic that getFormattedDate function returns formattedDate

Reply:
i named this function as 'formattedTime'. As for me, in that case, it best.